src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -22,111 +22,88 @@ the MIT License. See LICENSE in the project root for license information. -->
                              Style="{StaticResource RadioButtonSettingStyle}"/>
             </DataTemplate>
             <SettingsModel:IconPathConverter x:Key="IconSourceConverter"/>
+            <Style x:Key="LaunchSizeNumberBoxStyle" TargetType="muxc:NumberBox" BasedOn="{StaticResource NumberBoxSettingStyle}">
+                <Setter Property="SmallChange" Value="1"/>
+                <Setter Property="LargeChange" Value="10"/>
+                <Setter Property="Minimum" Value="1"/>
+            </Style>
         </ResourceDictionary>
     </Page.Resources>
 
     <ScrollViewer>
-        <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <!--Default Profile-->
-            <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                <ComboBox x:Uid="Globals_DefaultProfile"
+        <StackPanel>
+            <StackPanel Style="{StaticResource SettingsStackStyle}">

```

Comment:
You probably don't need this second `StackPanel`

Reply:
ScrollViewer only accepts one content item, so I had to put everything into a combined outer stackpanel